config/packer/vetafi-ubuntu-14.04.4-server-amd64.json

```diff
@@ -109,6 +127,11 @@
                 "provider": "virtualbox",
                 "version": "0.0.4"
             }
-        }]
+        },
+        {
+            "type": "atlas",
+            "artifact": "qwwqwwq/nodejs-mongodb-redis",
```

Comment:
I've noticed that there are a lot of personal folders in a lot of these paths. Can we make them a globally named path of sorts? This way, we can more confident things work/don't work based on personal local environments.